src/Subscriber.js

```diff
@@ -114,11 +114,20 @@ class Subscriber {
     }
 
     let timedOut = false
-    const timer = setTimeout(() => {
-      timedOut = true
-      this._logger.error('Timeout in Subscriber', this.name, request.data)
-      this._nack(channel, msg)
-    }, this._timeoutMs)
+    let timer
```

Comment:
You could make it simpler by keeping the existing code for the timer and only change the time parameter's value.

```js
let timeoutMs = typeof msg.timeOut === 'number' ? msg.timeOut : this._timeoutMs
timer = setTimeout(() => {
  timedOut = true
  this._logger.error('Timeout in Subscriber', this.name, request.data)
  this._nack(channel, msg)
}, timeoutMs)
```